pkg/profilecreator/profilecreator_test.go

```diff
@@ -10,6 +10,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
This file has merge conflicts, we need to rebase.

Reply:
Done